www/FirebaseAnalytics.js

```diff
@@ -49,24 +49,7 @@ module.exports = {
             exec(resolve, reject, PLUGIN_NAME, "setDefaultEventParameters", [defaults || {}]);
         });
     },
-    requestTrackingAuthorization: function(showInformation, title, message, buttonTitle) {
-        return new Promise(function(resolve, reject) {
-
-            if(showInformation) {
-                if (typeof title !== "string") {
-                    return reject(new TypeError("Title property name must be a string"));
-                }
-    
-                if (typeof message !== "string") {
-                    return reject(new TypeError("Message property value must be a string"));
-                }
-    
-                if (typeof buttonTitle !== "string") {
-                    return reject(new TypeError("Button title property value must be a string"));
-                }
-            }
-
-            exec(resolve, reject, PLUGIN_NAME, "requestTrackingAuthorization", [showInformation, title, message, buttonTitle]);
-        });
+    requestTrackingAuthorization: function(showInformation, title, message, buttonTitle, success, error) {
+        exec(success, error, 'FirebaseAnalytics', 'requestTrackingAuthorization', [showInformation, title, message, buttonTitle]);
```

Comment:
maybe use `PLUGIN_NAME` as it was? Just to keep all things accordingly.

Reply:
Yup, nice catch! Done